ui/app/components/app/connected-accounts-list/index.scss

```diff
@@ -0,0 +1,161 @@
+.connected-accounts-list {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  &__identicon {
+    margin-right: 8px;
+  }
+
+  &__account-name {
+    font-weight: bold;
+    font-size: 14px;
+    line-height: 20px;
+  }
+
+  &__primary {
```

Comment:
I suspect this class is no longer used. It's used below as well, alongside `&__account-status`